store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9586,15 +9587,20 @@ TODO: delete them permanently from here
   </desc>
 </attr>
 
-<attr id="2126" name="zimbraNetworkHSMNGEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" requiresRestart="mailbox" since="8.8.3">
+<attr id="2129" name="zimbraNetworkHSMNGEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" requiresRestart="mailbox" since="8.8.5">
```

Comment:
I am not sure if changing id will break upgrade or not. Can you please test 8.8.3 to 8.8.5 upgrade.

Reply:
If this is new attribute and not yet released, it should be fine.

Reply:
This is a new attribute, so it will be fine.